Language/Functions/Advanced IO/tone.adoc

```diff
@@ -72,6 +72,7 @@ If you want to play different pitches on multiple pins, you need to call `noTone
 
 [role="language"]
 * #LANGUAGE# link:../../analog-io/analogwrite[analogWrite()]
+* #LANGUAGE# link:../notone[noTone()]
```

Comment:
```suggestion
```
The website generation system normally automatically creates "See also" links for all the pages in the same folder. Unfortunately, this functionality is currently broken. I'm hoping this will soon be fixed.

If the developers did decide that the functionality can't be restored, I would welcome assistance in manually adding the links back to all the reference pages, but there is currently no indication that will be necessary.

Reply:
Oh, sure! I'll remove that now.